README.md

```diff
@@ -6,12 +6,12 @@ ancient) or geographical region (Africa-MiddleEast, Europe-Americas, South Asia,
 
 We offer two types of combined fonts:
 
-1. Time-based:  
+1. Time-based:
```

Comment:
Can you please keep the two trailing spaces as they are? It forces a line break in Markdown.

Same comment on line 14.

Reply:
Sorry totally forgot about that; I was trying to remove any trailing spaces that might have been created.
It should be fixed now!